content/swiftui/concepts/viewmodifier/terms/padding/padding.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.padding()'
+Description: 'Applies spacing around a view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.padding()`** modifier method applies padding or spacing around a [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+Text("This text has padding around it.")
+    .padding(edges, amount)
+```
+
+The `.padding()` modifier method uses the following optional parameters:
+
+- `edge` specifies which side(s) the padding should be applied to. The following options are possible:
+  - `.all` (default option): Applies padding to all edges of the view
+  - `.top`: Applies padding to the top edge of the view
+  - `.bottom`: Applies padding to the bottom edge of the view
+  - `.trailing`: Applies padding to the right edge of the view
+  - `.leading`: Applies padding to the left edge of the view
+  - `.vertical`: Applies padding to both the left and right edges of the view
+  - `.horizontal`: Applies padding to both the top and bottom edges of the view
```

Comment:
I think this may be easier to read in a table like with 2 columns. One for the parameter and another for the edges the padding is applied. This way, we don't have to keep writing "applies padding to..."

Reply:
I changed the list to a table.